runtime/src/accounts_db.rs

```diff
@@ -121,31 +122,31 @@ impl Versioned for (u64, AccountInfo) {
 }
 
 #[derive(Clone, Default, Debug)]
-pub struct AccountStorage(pub HashMap<Slot, SlotStores>);
+pub struct AccountStorage(pub DashMap<Slot, Arc<RwLock<SlotStores>>>);
```

Comment:
nits: how about adjusting the type alias `SlotStores` and hide `Arc<RwLock<_>>` here?